tests/test_utils/graphs.py

```diff
@@ -198,7 +198,12 @@ def example_graph_1_saliency_maps(feature_size=None):
 
 
 def example_graph_random(
-    feature_size=4, n_edges=20, n_nodes=6, n_isolates=1, is_directed=False
+    feature_size=4,
+    n_edges=20,
+    n_nodes=6,
+    n_isolates=1,
+    is_directed=False,
+    rand_feats=False,
```

Comment:
Since this is used rarely, I think it's ok to not be too brief here, and make it easier to understand what it's referring to when reading the code in isolation:

```suggestion
    random_features=False,
```

Another option would be to try always using random features (i.e. replacing the `np.ones` with `np.random.random` in the next line), and see if anything is currently relying on `1` features.